types/shared.d.ts

```diff
@@ -10,28 +10,28 @@ export declare type UIConfigOptions = {
 export declare type AppearanceOptions = {
   colorPrimary?: string;
   // Primary Button
-  colorPrimaryButtonBorder?: string;
-  colorPrimaryButtonBackground?: string;
-  colorPrimaryButtonText?: string;
+  buttonPrimaryColorBorder?: string;
+  buttonPrimaryColorBackground?: string;
+  buttonPrimaryColorText?: string;
   // Secondary Button
-  colorSecondaryButtonBorder?: string;
-  colorSecondaryButtonBackground?: string;
-  colorSecondaryButtonText?: string;
+  buttonSecondaryColorBorder?: string;
+  buttonSecondaryColorBackground?: string;
+  buttonSecondaryColorText?: string;
   // Link
-  colorPrimaryLinkText?: string;
-  colorSecondaryLinkText?: string;
+  actionPrimaryColorText?: string;
+  actionSecondaryColorText?: string;
   // Border
   colorBorder?: string;
   // Form
-  colorFormHighlightBorder?: string;
-  colorFormAccent?: string;
+  formHighlightColorBorder?: string;
+  formAccentColor?: string;
   // Text
   colorText?: string;
   colorSecondaryText?: string;
   // Background
   colorBackground?: string;
```

Comment:
Sorry to bump a merged request, but any reason why we have properties like `colorBackground` still, while offset has been changed to `offsetBackgroundColor` while even some have been changed to `buttonSecondaryColorBackground`. Not sure if its just me but the ordering of these properties still seem confusing to me. In these examples its hard for me to tell whether color or background should come first

Reply:
@nathanbullenmyob no problem, thank you for the feedback!

For properties associated with a component, like `badgeSuccessColorBackground` and `buttonSecondaryColorBackground`, we have made changes such that the variable name uses the component first.

We are aiming to keep our syntax consistent for more commonly used variables, such as: `colorPrimary`, `colorText`, and `colorBackground`.

It is also worth noting that the ordering of the variables does not impact how they are applied to the appearance of the embedded components. To see what the impact of these variables is, refer to our documentation at https://stripe.com/docs/connect/customize-connect-embedded-components